Gruntfile.js

```diff
@@ -62,14 +62,15 @@ module.exports = function(grunt) {
 
         watch: {
             files: ['js/**/*.js'],
```

Comment:
Because this is all JS related, I'd change this to:

```
watch: {
   js: {
      files: ['js/**/*.js'],
      tasks: ['concurrent', 'concat', 'uglify']
   }
}
```

So that we can add more targets later on (e.g. Sass)